src/lib/Scenes/Fair/Components/FairBooth/FairBooth.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box } from "@artsy/palette"
+import { FairBooth_show } from "__generated__/FairBooth_show.graphql"
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import { CaretButton } from "lib/Components/Buttons/CaretButton"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { FairBoothHeader } from "./Components/FairBoothHeader"
+
+interface Props {
+  show: FairBooth_show
+  onViewWorksPressed: () => void
+}
+
+export const FairBoothContainer = createFragmentContainer<Props>(
+  props => {
```

Comment:
Nice use of anonymous component. I think we should use this pattern for other relay containers.